packages/react-api/src/Api.tsx

```diff
@@ -232,11 +235,23 @@ async function getLightProvider (chain: string): Promise<ScProvider> {
 async function createApi (apiUrl: string, signer: ApiSigner, onError: (error: unknown) => void): Promise<Record<string, Record<string, string>>> {
   const types = getDevTypes();
   const isLight = apiUrl.startsWith('light://');
+  let provider;
 
   try {
-    const provider = isLight
-      ? await getLightProvider(apiUrl.replace('light://', ''))
-      : new WsProvider(apiUrl);
+    if (isLight) {
+      provider = await getLightProvider(apiUrl.replace('light://', ''));
+    } else if (islocalFork) {
```

Comment:
We can call `store.get('isLocalFork')` directly in here, and it would remove `isLocalFork` from global scope as well.

e.g `else if (store.get('isLocalFork'))`